concepts/basics/introduction.md

```diff
@@ -34,10 +34,10 @@ class Calculator {
 }
 ```
 
-Invoking a method is done by specifying its class and method name and passing arguments for each of the method's parameters.
+Invoking/calling a method is done by specifying its class and method name and passing arguments for each of the method's parameters.
 
 ```java
-int sum = new Calculator().add(1, 2);
+int sum = new Calculator().add(1, 2);  // here the  "add" method has been called to perform the task of addition 
```

Comment:
This suggestion is mainly to reduce the width (reducing chance of needing side scrolling)

```suggestion
// call "add" method to perform addition 
int sum = new Calculator().add(1, 2);  
```